client/src/components/ReviewPhotosCarousel.jsx

```diff
@@ -2,14 +2,17 @@ import React from 'react';
 import styles from './css/reviewPhotosCarouselStyles.css';
 
 const ReviewPhotosCarousel = (props) => {
+  const minIndex = (props.page - 1) * 5;
```

Comment:
interesting that you chose to use to use a non-zero based page numbering system (I assume you start at page-1).  I'm not sure how relevant this is, but something in my memory banks is telling me that it's a good practice to maintain the index-base a language has (in javascript, that is 0), where it make sense (for logic and math reasons).

Reply:
NBD either way.

Reply:
yeah I can't see why it would matter too much either way 🤔 I figured if you wanted to display the page number it would make more sense for it to be 1 indexed, but at the same time you could also have it 0-indexed and just add 1 to it for the display...if you find something that explains what's best I'd love to see it!